.github/workflows/call.deploy-subgraph.yml

```diff
@@ -28,10 +28,39 @@ on:
         required: false
 
 jobs:
+  show-contexts:
+    name: Show Contexts
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Show contexts
+        run: |
+          echo github.event_name: ${{ github.event_name }}
+          echo github.sha: ${{ github.sha }}
+          echo github.repository: ${{ github.repository }}
+          echo github.ref: ${{ github.ref }}
+          echo github.head_ref: ${{ github.head_ref }}
+          echo github.base_ref: ${{ github.base_ref }}
+
+  build-and-test-local-subgraph:
```

Comment:
I appreciate the meticulousness, but repeating "key" and "name" with the same information is not necessarily adding anything but noise. I think it's worth to be conscious of not adding syntactic noise too casually.

Reply:
"build-and-test-local-subgraph:" is almost conflict with "name: Build and Test Subgraph (Release Branch)" on the surface, so not sure.